pom.xml

```diff
@@ -113,7 +114,6 @@
       <groupId>io.cdap.cdap</groupId>
       <artifactId>cdap-common</artifactId>
       <version>${cdap.version}</version>
-      <scope>provided</scope>
```

Comment:
do you know what this is being used for? cdap-common is supposed to be a common module for the CDAP project, not sure why it would be a dependency for a plugin.

Reply:
It's also not provided by CDAP at runtime, so if it was actually required before, I don't see how it could have worked.

Reply:
We are using a few CDAP common classes in tests and in the `ByteBuffers` class for https://github.com/data-integrations/kafka-plugins/blob/0172d71ab214be34feba02ed04c2a909d9dc60f2/kafka-plugins-client/src/main/java/io/cdap/plugin/batch/source/KafkaBatchSource.java

Reply:
Can we replace that method with Bytes.toBytes() (this is in cdap-api-common) and change this to test scope? cdap-common pulls in a ton of extra dependencies.

Doesn't have to be in this PR.

Reply:
Let me see if I can pull this dependency off from our test classes completely.